packages/client/package.json

```diff
@@ -137,7 +137,8 @@
     }
   },
   "dependencies": {
-    "@prisma/engines-version": "3.16.0-43.81a0ee489e5f8c8ce98440028ea9050092430503"
+    "@prisma/engines-version": "3.16.0-43.81a0ee489e5f8c8ce98440028ea9050092430503",
+    "ts-toolbelt": "9.6.0"
```

Comment:
Why bundling doesn't work? (Until now we bundled all deps)

Reply:
Bundling currently produces broken types. It's also a huge library, and bundling would include a lot of unused types. It would be very clumsy to copy the sources of eg. `O.Optional`.

Reply:
The other alternative is to not correctly type for now and remove the dependency.

Reply:
So only for that line here?
https://github.com/prisma/prisma/pull/13811/files#diff-09784fc0113a735572cff48990efa35434fb6ec6ec8876845ec9cf423642e5fdR214

Reply:
Correct

Reply:
Replaced with `Omit` as per your suggestion 👍